WooCommerce/Classes/Tools/Developer/DeveloperEmailChecker.swift

```diff
@@ -0,0 +1,11 @@
+import Foundation
+
+struct DeveloperEmailChecker {
```

Comment:
Being a struct, it should not be that expensive to create an instance of this checker. On the one hand, the checker is a function, with a clear input and output, so it makes sense to declare it as static, but on the other hand we would not be able to pass an instance around if needed (which I don't know if it's an actual issue).

So, I'm curious, what made you decide declaring it as static?

Reply:
I generally prefer not using static whenever possible, like you said it's easier to pass around and DI, even though the function is purely functional. I picked static here after checking a few helper methods in the codebase that use static functional methods (like `CoreDataIterativeMigrator`, `SharingHelper`) and thought it's the convention. I can update to instance if we (Woo iOS devs @mindgraffiti) are all good with using instance methods for helpers like this case :)

Reply:
Many of the Helpers you'll see in our codebase originally came from WPiOS, where (I believe) the convention was to make everything static. `CoreDataIterativeMigrator` is a port from WPiOS, but was converted to Swift, and `SharingHelper` also came from WPiOS.

If you check out our more recent Helpers, like `NSDecimalNumber+Helpers` or `Dictionary+Woo`, you'll see that we preferred instance methods. **However** all I can do is offer the historical context on our previous decisions. If you find areas to improve, by all means, we're open to suggestions :)

tl;dr: I agree with @ctarda and defer to his recommendation because it's not my area of expertise.

Reply:
Thanks for the context and pointers! I updated this static method to instance method in https://github.com/woocommerce/woocommerce-ios/pull/1043/commits/35950ccbfb6fed5af89125306428ce7c574bb526 - I'd like to follow team convention on utility methods whether it's static or instance :)